src/features/projectsV2/ProjectDetails/components/SinglePlantInfo.tsx

```diff
@@ -0,0 +1,48 @@
+import PlantInfoCard from './microComponents/PlantInfoCard';
+import { formatHid } from '../../../../utils/projectV2';
+import styles from '../styles/PlantLocationInfo.module.scss';
+import { useTranslations } from 'next-intl';
+import {
+  PlantLocation,
+  SamplePlantLocation,
+} from '../../../common/types/plantLocation';
+import getImageUrl from '../../../../utils/getImageURL';
+import TreeMapperBrand from './microComponents/TreeMapperBrand';
+
+interface Props {
+  plantData: SamplePlantLocation | PlantLocation | null;
+}
+
+const SinglePlantInfo = ({ plantData }: Props) => {
+  const t = useTranslations('ProjectDetails');
```

Comment:
Make a minor change to call this SinglePlantLocationInfo

Reply:
Since the sidebar only displays plant information and not location information, I believe that` 'SinglePlantInfo'` is the correct name for the component